CONTRIBUTING.md

```diff
@@ -237,6 +237,12 @@ $ git commit --amend
 $ git push --force-with-lease origin my-branch
 ```
 
+When the commits in your Pull Request get landed, they will be squashed into
+one commit, with metadata added to the commit message (including links to the
```

Comment:
I think this one could be a good example. https://github.com/nodejs/node/pull/2921